### PR TITLE
Add missing C# samples for the "Inspector plugins" page

### PR DESCRIPTION
**Pull request by seclerp**
Adding missing C# samples for the "Plugins > Inspector plugins" page makes it simpler to get started using the C# codebase.

The code below is dev-tested.

I also added a small DRY-compliant change to GDScript samples.

### CODE REVIEW

**Review by raulsntos — CHANGES_REQUESTED** [review]
Thanks for the contribution. The C# code needs to be surrounded by `#if TOOLS` (see https://github.com/godotengine/godot-docs/blob/dc20b238e3256a5a6abbf5b1bf6c0f5cf035c6e1/tutorials/plugins/editor/making_plugins.rst for an example) because editor classes (like `EditorPlugin`) are not available on exported games.

File: tutorials/plugins/editor/inspector_plugins.rst [hunk 1]
```diff
@@ -179,8 +231,72 @@ followed by ``set_bottom_editor()`` to position it below the name.
...
+
+        public RandomIntEditor()
+        {
+            // Add the control as a direct child of EditorProperty node.
+            AddChild(_propertyControl);
+            // Make sure the control is able to retain the focus.
+            AddFocusable(_propertyControl);
+            // Setup the initial state and connect to the signal to track changes.
+            RefreshControlText();
+            _propertyControl.Connect("pressed", this, nameof(OnButtonPressed));
```
Comment (raulsntos) [inline tutorials/plugins/editor/inspector_plugins.rst:262]: I forgot this is `master`'s documentation. The connecting syntax has changed and should now be:

```suggestion
            _propertyControl.Pressed += OnButtonPressed;
```

**Review by raulsntos — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.4 and 3.5.

**Pull request merged**